databases/jdp/database.go

```diff
@@ -222,14 +242,38 @@ func creditFromIdAndMetadata(id string, md Metadata) credit.CreditMetadata {
 	return crd
 }
 
-func trimFileSuffix(filename string) string {
-	for _, suffix := range supportedSuffixes {
-		trimmedFilename, trimmed := strings.CutSuffix(filename, "."+suffix)
-		if trimmed {
-			return trimmedFilename
+// creates a Frictionless DataResource-savvy name for a file:
+// * the filename is down-cased
+// * all periods are replaced with dashes
```

Comment:
are there any other weird characters? I usually find myself writing a generic function that converts any series of non-`[a-zA-Z0-9_]` characters into a single `_`